app/invocation/invocation_overview.tsx

```diff
@@ -204,6 +208,16 @@ export default class InvocationOverviewComponent extends React.Component<Props>
               {this.props.model.getCache()}
             </Link>
           )}
+          {hasCapabilityMetadata && (missingActionCacheWrite || missingCASWrite) && (
+            <div className="detail" title="This invocation's API key could not write to the cache">
+              <PencilOff className="icon" />
+              {missingActionCacheWrite && missingCASWrite
+                ? "Remote Cache writes disabled (AC and CAS)"
+                : missingActionCacheWrite
+                  ? "Action Cache writes disabled"
```

Comment:
If we're willing to abbreviate "AC" in the other labels, may as well abbreviate it here to make the detail a bit more compact.

```suggestion
                  ? "AC writes disabled"
```